### PR TITLE
Switcher,speeduper,tuner,teleporter mixed to one button

### PR DESCRIPTION
**Pull request by BannZay**

### CODE REVIEW

File: src/game/editor/editor.cpp [hunk 1]
```diff
@@ -1178,43 +1178,55 @@ void CEditor::DoToolbar(CUIRect ToolBar)
...
-			static int s_TunePopupID = 0;
-			UiInvokePopupMenu(&s_TunePopupID, 0, UI()->MouseX(), UI()->MouseY(), 120, 23, PopupTune);
+
+			TB_Bottom.VSplitLeft(5.0f, &Button, &TB_Bottom);
+			TB_Bottom.VSplitLeft(60.0f, &Button, &TB_Bottom);
+			{
+				typedef int(*popup)(CEditor *peditor, CUIRect View);
+				popup p;
+				static int layerButton = 0;
+				char *buttonName="Modifier";
```
Comment (def-) [inline src/game/editor/editor.cpp]: spaces missing

**Comment by def-** [comment]
Thanks.

File: src/game/editor/editor.cpp [hunk 2]
```diff
@@ -1178,43 +1178,55 @@ void CEditor::DoToolbar(CUIRect ToolBar)
...
+				float height;
```
Comment (def-) [inline src/game/editor/editor.cpp]: A default value would be nice to get rid of compiler warning:

    src/game/editor/editor.cpp:1227:88: warning: ‘height’ may be used uninitialized in this function [-Wmaybe-uninitialized]

**Comment by def-** [comment]
See my fix for reference of code style in the future, there were also some minor bugs: https://git.io/vr2n7

**Comment by BannZay** [comment]
I will, thank you

**Pull request merged**